truss/cli/cli.py

```diff
@@ -417,6 +435,44 @@ def deploy(
         )
 
 
+@chains.command(name="init")  # type: ignore
+@click.option(
+    "--target_directory",
+    type=str,
+    required=False,
+    help="Name of the chain to be deployed, if not given, the user will be prompted for a name",
+)
+@error_handling
+def chains_init(
+    target_directory: Optional[str],
+) -> None:
+    """
+    Initializes a chains project with hello.py
+    """
+    FILENAME = "main.py"
+    if not target_directory:
+        target_directory = input("Enter the target directory for the chain: ")
```

Comment:
Let's use the inquirer library like we do here: https://github.com/basetenlabs/truss/blob/b8fcdc390b18b8b57cc347467053c06d2c7ab3a3/truss/cli/cli.py#L530

Reply:
don't quite understand what you mean here - would love to chat about it

Reply:
sync'd offline - leveraging the inquirer library here for text input